pages/topics/client-distribution.mdx

```diff
@@ -0,0 +1,341 @@
+
+</Alert>
+
+###### Query String Params
+
+| Name        | Type   | Description                                                                                  |
+| ----------- | ------ | -------------------------------------------------------------------------------------------- |
+| platform    | string | The [platform](#desktop-platform-type) to get the installer for                              |
+| format? ^1^ | string | The [executable format](#desktop-executable-format) to get the installer for (default `deb`) |
+
+^1^ Only applicable to the [Linux platform](#desktop-platform-type).
+
+<RouteHeader method="GET" url="/updates/{release_channel}" deprecated>
+  Get Application Updates
+</RouteHeader>
+
+Returns information about the latest application update for the provided [release channel](#desktop-release-channel) and selected platform.
+
+<Alert type="warn">
+
+For certain platforms, this endpoint is deprecated. It is in the process of being replaced by [Get Latest Distributed Application Manifest](#get-latest-distributed-application-manifest).
+
+</Alert>
+
+<Alert type="info">
+
+All parameters to this endpoint are optional.
+
+</Alert>
+
+###### Query String Params
+
+| Name      | Type   | Description                                                                          |
+| --------- | ------ | ------------------------------------------------------------------------------------ |
+| platform? | string | The [platform](#desktop-platform-type) to get update information for (default `osx`) |
+
+###### Response Body
+
+| Name       | Type              | Description                            |
+| ---------- | ----------------- | -------------------------------------- |
+| name       | string            | The latest host version                |
+| pub_date   | ISO8601 timestamp | When the update was published          |
+| url? ^1^   | string            | The URL to the corresponding installer |
+| notes? ^1^ | string            | Any extra notes for the update         |
+
+^1^ Only provided if auto updates are available for the selected platform.
+
+###### Example Response
+
+```json
+{
+  "name": "0.0.75",
+  "pub_date": "2023-07-05T17:16:10",
+  "url": "https://dl-ptb.discordapp.net/apps/osx/0.0.75/DiscordPTB.zip",
+  "notes": ""
+}
+```
+
+<RouteHeader method="GET" url="/modules/{release_channel}/versions.json" deprecated>
+  Get Native Module Versions
+</RouteHeader>
+
+Returns a mapping of module names to integer versions representing the native module versions for the provided [release channel](#desktop-release-channel) and selected platform, if found.
+
+{/* TODO: How the fuck is this used */}
+
+<Alert type="warn">
+
+For certain platforms, this endpoint is deprecated. It is in the process of being replaced by [Get Latest Distributed Application Manifest](#get-latest-distributed-application-manifest).
```

Comment:
On Windows it is not "deprecated", it has been completely unused for years and older clients using it would not work at all.

Reply:
How do you think I can word this better?